src/fides/lib/oauth/api/urn_registry.py

```diff
@@ -0,0 +1,16 @@
+# Prefix
```

Comment:
We could move this into the larger urn_registry now that everything is together, or the ones from ops could move here.

Reply:
I've created a new issue for further refactoring, so it'll be easier to track these smaller changes when reviewing